dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -44,6 +44,17 @@ def color_for_status(status)
 Given /^I am on "([^"]*)"$/ do |url|
   url = replace_hostname(url)
   @browser.navigate.to "#{url}"
+  install_js_error_recorder
+end
+
+def install_js_error_recorder
+  @browser.execute_script(<<-JS
+  window.onerror = function (msg) {
+    window.detectedJSErrors = window.detectedJSErrors || [];
+    window.detectedJSErrors.push(msg);
+  };
```

Comment:
Ah that's right, like we do in https://github.com/code-dot-org/code-dot-org/blob/staging/code-studio/src/js/code-studio.js#L63-L63 — will update